Cargo.toml

```diff
@@ -18,12 +16,16 @@ version = "0.1.0"
 edition.workspace = true
 rust-version.workspace = true
 default-run = "fish"
+license = "GPL-2.0-only AND BSD-3-Clause AND ISC AND LGPL-2.0-or-later AND MIT AND PSF-2.0"
```

Comment:
Can you please share how you got this? fish itself is released under GPL 2.0

Reply:
It was pulled from https://src.fedoraproject.org/rpms/fish/blob/rawhide/f/fish.spec#_33

Reply:
Ah interesting, thank you! Some of those dependencies are no more so we can rework that. Thank you for tackling this!

Reply:
Had a quick check and updated this with an explanation of where things are coming from. Shoutout to the fedora packaging team for doing this initially :)

I'm unsure about `doc_src/python_docs_theme/`, as the files in there reference the BSD license, but `license.rst` mentions the PSF-2.0

Reply:
The docs theme is PSF-2 per https://pypi.org/project/python-docs-theme/2020.1/ but there are some Sphinx bits included from them that are BSD-licensed.